src/main/test/groovy/scrumweb/user/account/controller/AuthenticationControllerTest.groovy

```diff
@@ -1,18 +1,23 @@
 package scrumweb.user.account.controller
```

Comment:
usuń te testy, bo nie działają, i kiedy będziemy chcieli odpalić wszystkie testy, to nie potrafimy to zrobić xD